### PR TITLE
Test suite process update and dependencies/equality bug fixes

### PR DESCRIPTION
**Pull request by lvenerosy**
- benchmarking test suite update
- package.json wrong dependencies fix
- equality relaxation fix
- version update to 0.4.4 in package.json

### CODE REVIEW

**Comment by lvenerosy** [comment]
The "equality relaxation fix" is critical enough to publish a new version (I already modified the version in the package.json).

File: package.json [hunk 1]
```diff
@@ -16,14 +17,14 @@
...
     "benchmark": "*",
+    "mocha": "3.1.2",
     "grunt": "*",
     "grunt-browserify": "^4.0.1",
     "grunt-contrib-jshint": "~0.9",
     "grunt-contrib-uglify": "*",
     "grunt-jsbeautifier": "*",
     "grunt-mocha-test": "*",
-    "walk": "*",
-    "fs": "*"
```
Comment (bchevalier) [inline package.json]: it's not necessary to require the "fs" module?

File: package.json [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-  "version": "0.4.3",
+  "version": "0.4.4",
   "private": false,
   "authors": [
     "Justin W. Wolcott"
   ],
   "contributors": [
-    "Brice Chevalier <bchevalier@wizcorp.jp>"
+    "Brice Chevalier <bchevalier@wizcorp.jp>",
+    "Loïc Venerosy <lvenerosy@wizcorp.jp>"
```
Comment (JWally) [inline package.json]: Been meaning to do that!

File: package.json (hunk 1 above)
Comment (lvenerosy) [inline package.json]: Both browserify and the test suite work without requiring fs.

**Comment by bchevalier** [comment]
looks good to me

**Pull request merged**